pom.xml

```diff
@@ -33,7 +33,7 @@
     <parent>
         <groupId>com.forgerock.openbanking</groupId>
         <artifactId>forgerock-openbanking-starter-parent</artifactId>
-        <version>1.2.7-SNAPSHOT</version>
+        <version>1.2.6-SNAPSHOT</version>
         <relativePath /> <!-- lookup parent from repository -->
```

Comment:
The parent must be 1.2.7-SNAPSHOT that is the current version on master

Reply:
Fixed